src/navigation/Navigator.tsx

```diff
@@ -28,26 +28,32 @@ export const styles = StyleSheet.create({
     borderBottomColor: COLORS.lunesBlackUltralight,
     borderBottomWidth: 1
   },
-  title: {
-    color: COLORS.lunesBlack,
-    fontFamily: 'SourceSansPro-SemiBold',
-    fontSize: wp('4%'),
-    textTransform: 'uppercase',
-    fontWeight: '600',
-    marginLeft: 15,
-    letterSpacing: 0.4
+  headerRightContainer: {
+    paddingRight: 15,
+    flexShrink: 1
   },
-  headerLeft: {
-    paddingLeft: 15,
-    flexDirection: 'row',
-    alignItems: 'center',
-    zIndex: 100
-  },
-  headerRight: {
-    paddingRight: 15
+  headerLeftContainer: {
+    flexGrow: 3
   }
 })
 
+export const NavigationTitle = styled.Text`
```

Comment:
Should we put styles in separate file?
I think in this case it make sense to reuse it f.e. in ExerciseHeader

Reply:
I think it makes sense if it is actually used in multiple places. Then just create a `NavigationTitle` component file

Reply:
hm the problem is that this is not a real component but a style wrapper for a component that i want to share.
i thought about sth like this f.e. having a folder like "test" but "styles" and create Navigator inside with its styles
https://blog.cloudboost.io/separate-your-code-with-styled-components-ec4fd1ee3ef8

Reply:
Its usable as a component though (and technically is a react component), isn't it? So no problem creating a separate file for it :)